### DISCUSSION TITLE
How do I override the random source?

### DISCUSSION THREAD

**Discussion by CMCDragonkai** [panva/jose#453] [Q&A]
Is there a way to override the random source being used?

I see that there's a `src/runtime/random.ts` file being used on different platforms.

However I'd like to be able to override it so I can centralise my random source across multiple libraries.

Other libraries do expose the ability to monkey patch globally. But it appears the random module exported as a default, which makes it difficult to import a module and just reassign the random function.

**Comment by panva** [comment]
> Is there a way to override the random source being used?

There is not.
